platform/android/java/lib/src/com/google/android/vending/expansion/downloader/impl/DownloadNotification.java

```diff
@@ -29,7 +29,6 @@
 import android.app.NotificationManager;
 import android.app.PendingIntent;
 import android.content.Context;
-import android.os.Build;
```

Comment:
Is this needed? This is a modification of thirdparty code, so it should be documented as I did for the other changes with `// -- GODOT start --` and `// -- GODOT end --` comments, and adding it to the `.patch` file that describes these changes.